src/components/navigation-header/navigation-header.module.css

```diff
@@ -22,8 +22,8 @@
   width: 100%;
 
   @media (--dev-dot-tablet-down) {
-    padding-bottom: 16px;
-    padding-top: 16px;
+    padding-bottom: 14px;
+    padding-top: 14px;
```

Comment:
This is intended to fix the issue on all product logo pages. Previously nav bar height on mobile was `72px`, it should now be `68px`. Thanks Ashlee for tracking this down! 🙌